usaspending_api/etl/elasticsearch_loader_helpers/delete_data.py

```diff
@@ -115,6 +140,17 @@ def delete_docs_by_unique_key(client: Elasticsearch, key: str, value_list: list,
     if not index:
         raise RuntimeError("index name must be provided")
 
+    if use_aliases:
+        index = f"{index}-*"
```

Comment:
I'm not following the usecase of `use-aliases`. Deletes cannot occur on an alias that points to multiple indexes. (https://discuss.elastic.co/t/delete-a-document-from-an-alias/55789/2). Additionally, a delete function that uses a wildcard for the name seems to be asking for trouble.

Reply:
I'll have to look into this. During unit testing, the index passed in _was_ the alias here, which caused complications until this was added -- following the pattern that was in the other delete method `_delete_from_es`:

https://github.com/fedspendingtransparency/usaspending-api/blob/45429fa55fc2984b58ba06f2ea8bd8f915094aa4/usaspending_api/etl/elasticsearch_loader_helpers/delete_data.py#L58-L59

Reply:
Figured out how to force it to use the write alias (like for incremental load) and therefore needs no wildcard. Removed this.